src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -432,6 +438,10 @@ public static JsonObjectBuilder json(DatasetVersion dsv, List<String> anonymized
                 .add("publicationDate", dataset.getPublicationDateFormattedYYYYMMDD())
                 .add("citationDate", dataset.getCitationDateFormattedYYYYMMDD());
 
+        if(latestDsv != null) {
+            bld.add("latestVersionPublishingStatus", latestDsv.getVersionState().name()).add("versionNote", latestDsv.getVersionNote());
```

Comment:
What is the purpose of `versionNote` here? I'm asking to know better the intention here.

If you are referring to the latest version note, I would rename it to `latestVersionNote` to avoid confusion.

Reply:
I am not fully aware of the versionNote, this is how the regular version is attached to the JSON and since the latestVersion is retrieved in the same way I decided to leave it exactly than what is on 431 probably @qqmyers can enlighten us a little bit more here 😆

Reply:
Resolved after our latest slack call, thanks! 👍🏼